File: src/main/java/vserver/HttpMethod.java
package vserver;

public enum HttpMethod {
    GET, POST, PUT, DELETE, HEAD, OPTIONS, PATCH, CONNECT, TRACE;

    public static final HttpMethod[] ALL_METHODS = new HttpMethod[]{
        GET, POST, PUT, DELETE, HEAD, OPTIONS, PATCH, CONNECT, TRACE
    };
}

File: lib/src/main/java/io/vproxy/lib/http/HttpMethod.kt
package io.vproxy.lib.http

enum class HttpMethod {
  GET, POST, PUT, DELETE, HEAD, OPTIONS, PATCH, CONNECT, TRACE;

  companion object {
    @JvmField
    val ALL_METHODS = arrayOf(GET, POST, PUT, DELETE, HEAD, OPTIONS, PATCH, CONNECT, TRACE)
  }
}
